### ISSUE TITLE
v0.17.0 breaks compatibility

### ISSUE THREAD

**Issue by akfaew** [xanzy/go-gitlab#620]
My tests pass with v0.16.1 but break with v0.17.0

`json: cannot unmarshal object into Go struct field .updated_by_id of type []int`

This comes from `gitlab.ParseWebhook(gitlab.WebhookEventType(r), payload)`

The payload is:
```
{                                                                                                                                                                                                                                                                                                                              
   "object_attributes" : {                                                                                                                                                                                                                                                                                                     
      "created_at" : "2019-03-29 08:35:11 UTC",                                                                                                                                                                                                                                                                                
      "state" : "opened",                                                                                                                                                                                                                                                                                                      
      "milestone_id" : null,                                                                                                                                                                                                                                                                                                   
      "title" : "Test MR",                                                                                                                                                                                                                                                                                                     
      "merge_params" : {                                                                                                                                                                                                                                                                                                       
         "force_remove_source_branch" : "0"                                                                                                                                                                                                                                                                                    
      },                                                                                                                                                                                                                                                                                                                       
      "action" : "update",                                                                                                                                                                                                                                                                                                     
      "merge_user_id" : null,                                                                                                                                                                                                                                                                                                  
      "updated_by_id" : 123,                                                                                                                                                                                                                                                                                                   
      "human_total_time_spent" : null,                                                                                                                                                                                                                                                                                         
      "time_estimate" : 0,                                                                                                                                                                                                                                                                                                     
      "merge_commit_sha" : null,                                                                                                                                                                                                                                                                                               
      "last_edited_by_id" : null,                                                                                                                                                                                                                                                                                              
      "source_project_id" : 123,                                                                                                                                                                                                                                                                                               
      "merge_error" : null,                                                                                                                                                                                                                                                                                                    
      "last_commit" : {                                                                                                                                                                                                                                                                                                        
         "id" : "xxx",                                                                                                                                                                                                                                                                                                         
         "message" : "e\n",                                                                                                                                                                                                                                                                                                    
         "author" : {                                                                                                                                                                                                                                                                                                          
            "name" : "XXX YYY",                                                                                                                                                                                                                                                                                                
            "email" : "xxx@example.com"                                                                                                                                                                                                                                                                                        
         },                                                                                                                                                                                                                                                                                                                    
         "timestamp" : "2019-03-29T06:16:18Z",                                                                                                                                                                                                                                                                                 
         "url" : "https://gitlab.com/xxx/subgroup/xxx/commit/xxx"                                                                                                                                                                                                                                                              
      },                                                                                                                                                                                                                                                                                                                       
      "url" : "https://gitlab.com/xxx/subgroup/xxx/merge_requests/12",                                                                                                                                                                                                                                                         
      "author_id" : 123,                                                                                                                                                                                                                                                                                                       
      "assignee_id" : null,                                                                                                                                                                                                                                                                                                    
      "last_edited_at" : null,                                                                                                                                                                                                                                                                                                 
      "source_branch" : "e",                                                                                                                                                                                                                                                                                                   
      "target_branch" : "master",                                                                                                                                                                                                                                                                                              
      "description" : "",                                                                                                                                                                                                                                                                                                      
      "work_in_progress" : false,                                                                                                                                                                                                                                                                                              
      "human_time_estimate" : null,                                                                                                                                                                                                                                                                                            
      "source" : {
         "default_branch" : "master",
         "git_ssh_url" : "git@gitlab.com:xxx/subgroup/xxx.git",
         "avatar_url" : null,
         "homepage" : "https://gitlab.com/xxx/subgroup/xxx",
         "ssh_url" : "git@gitlab.com:xxx/subgroup/xxx.git",
         "git_http_url" : "https://gitlab.com/xxx/subgroup/xxx.git",
         "url" : "git@gitlab.com:xxx/subgroup/xxx.git",
         "path_with_namespace" : "xxx/subgroup/xxx",
         "web_url" : "https://gitlab.com/xxx/subgroup/xxx",
         "namespace" : "Subgroup",
         "description" : "",
         "http_url" : "https://gitlab.com/xxx/subgroup/xxx.git",
         "ci_config_path" : null,
         "name" : "xxx",
         "id" : 123,
         "visibility_level" : 0
      },
      "total_time_spent" : 0,
      "head_pipeline_id" : null,
      "id" : 123,
      "merge_status" : "can_be_merged",
      "target_project_id" : 123,
      "iid" : 12,
      "merge_when_pipeline_succeeds" : false,
      "updated_at" : "2019-03-29 08:35:50 UTC",
      "target" : {
         "default_branch" : "master",
         "git_ssh_url" : "git@gitlab.com:xxx/subgroup/xxx.git",
         "avatar_url" : null,
         "homepage" : "https://gitlab.com/xxx/subgroup/xxx",
         "ssh_url" : "git@gitlab.com:xxx/subgroup/xxx.git",
         "git_http_url" : "https://gitlab.com/xxx/subgroup/xxx.git",
         "url" : "git@gitlab.com:xxx/subgroup/xxx.git",
         "path_with_namespace" : "xxx/subgroup/xxx",
         "web_url" : "https://gitlab.com/xxx/subgroup/xxx",
         "namespace" : "Subgroup",
         "description" : "",
         "http_url" : "https://gitlab.com/xxx/subgroup/xxx.git",
         "ci_config_path" : null,
         "name" : "xxx",
         "id" : 123,
         "visibility_level" : 0
      }
   },
   "labels" : [],
   "user" : {
      "name" : "XXX YYY",
      "username" : "xxx",
      "avatar_url" : "https://secure.gravatar.com/avatar/xxx?s=80&d=identicon"
   },
   "event_type" : "merge_request",
   "repository" : {
      "description" : "",
      "homepage" : "https://gitlab.com/xxx/subgroup/xxx",
      "url" : "git@gitlab.com:xxx/subgroup/xxx.git",
      "name" : "xxx"
   },
   "project" : {
      "id" : 123,
      "visibility_level" : 0,
      "ci_config_path" : null,
      "name" : "xxx",
      "http_url" : "https://gitlab.com/xxx/subgroup/xxx.git",
      "namespace" : "Subgroup",
      "description" : "",
      "web_url" : "https://gitlab.com/xxx/subgroup/xxx",
      "path_with_namespace" : "xxx/subgroup/xxx",
      "ssh_url" : "git@gitlab.com:xxx/subgroup/xxx.git",
      "git_http_url" : "https://gitlab.com/xxx/subgroup/xxx.git",
      "url" : "git@gitlab.com:xxx/subgroup/xxx.git",
      "homepage" : "https://gitlab.com/xxx/subgroup/xxx",
      "avatar_url" : null,
      "git_ssh_url" : "git@gitlab.com:xxx/subgroup/xxx.git",
      "default_branch" : "master"
   },
   "object_kind" : "merge_request",
   "changes" : {
      "updated_by_id" : {
         "current" : 123,
         "previous" : null
      },
      "updated_at" : {
         "current" : "2019-03-29 08:35:50 UTC",
         "previous" : "2019-03-29 08:35:11 UTC"
      }
   }
}
```

**Comment by manland** [comment]
related to my PR #615

**Comment by svanharmelen** [comment]
This is fixed by #615
